presto-spark-base/src/main/java/com/facebook/presto/spark/BroadcastStorage.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.spark;
+
+/**
+ * BroadcastStorage specifies contract for specifying underlying file systems used for
```

Comment:
This comment is too cryptic. What is this interface for? Do we need it? 

CC: @shrinidhijoshi @vermapratyush

Reply:
Yeah. I believe something like this will be needed to generate the correct file paths during broadcast write. This will handle 
1. generating a basepath for that shuffle stage `QueryId.StageId.StageExecutionId` 
2. add a prefix of local path `/tmp/shuffle_data` ? or a prefix of the blob storage `ws://pos_prod/` depending on what's being used

Reply:
If my understanding is correct @singcha could you please update the comments to be descriptive like above ?

Reply:
Removed this interface and derived class. This is now specified with config